ui/cypress/e2e/dashboardsView.test.ts

```diff
@@ -60,6 +60,39 @@ describe('Dashboard', () => {
     cy.getByTestID('cell--view-empty').should('have.length', 1)
   })
 
+  // fix for https://github.com/influxdata/influxdb/issues/15239
+  it('retains the cell content after canceling an edit to the cell', () => {
+    cy.get('@org').then(({id: orgID}: Organization) => {
+      cy.createDashboard(orgID).then(({body}) => {
+        cy.fixture('routes').then(({orgs}) => {
+          cy.visit(`${orgs}/${orgID}/dashboards/${body.id}`)
+        })
+      })
+    })
+
+    // Add an empty celly cell
+    cy.getByTestID('add-cell--button').click()
+    cy.getByTestID('save-cell--button').click()
+    cy.getByTestID('cell--view-empty').should('be.visible')
+
+    let cellContent
+    cy.getByTestID('cell--view-empty')
+      .invoke('text')
+      .then(cellContent => {
```

Comment:
@hoorayimhelping , why're we calling the parameter that's being passed in the `then` the same thing as the variable that's being instantiated on line 78:

https://github.com/influxdata/influxdb/pull/15590/files#diff-33b992b7a030a90957e222522b90a778R78

Reply:
oops, didn't mean to leave `let cellContent` in there. i bet lint will complain now. good catch!